app/external_services/github_repos_api.rb

```diff
@@ -0,0 +1,15 @@
+class GithubReposApi
```

Comment:
Any reason why we can't have this service under `services`, with all the other service classes?

Reply:
the main reason is that all the services seems to have a pattern, they extends from `BaseService` and its entry points are the method `call`, for classes like this is probably that we will need to use methods for differents type of requests (GET some uri, POST another, etc).